components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/constant/ActionMgtConstants.java

```diff
@@ -23,8 +23,10 @@
  */
 public class ActionMgtConstants {
 
-    public static final String URI_ATTRIBUTE = "uri";
-    public static final String AUTHN_TYPE_ATTRIBUTE = "authnType";
+    public static final String URI_PROPERTY = "uri";
+    public static final String AUTHN_TYPE_PROPERTY = "authnType";
+    public static final String PASSWORD_SHARING_FORMAT_PROPERTY = "passwordSharingFormat";
```

Comment:
This constant should not come to the actions core. It's specific to a particular extension